### PR TITLE
Improve _tril() and _triu() with an ElementwiseKernel

### PR DESCRIPTION
**Pull request by rezoo**
This small PR speedups `_tril()` and `_triu()` in `cupy/linalg/util.py` with the ElementwiseKernel.

### CODE REVIEW

**Comment by okuta** [comment]
jenkins, test this please.

File: cupy/linalg/util.py [hunk 1]
```diff
@@ -26,18 +26,16 @@ def _assert_nd_squareness(*arrays):
 
 
 def _tril(x, k=0):
-    m, n = x.shape
-    u = cupy.arange(m).reshape(m, 1)
-    v = cupy.arange(n).reshape(1, n)
-    mask = v - u <= k
-    x *= mask
+    cupy.ElementwiseKernel(
```
Comment (okuta) [inline cupy/linalg/util.py]: Cloud you cache this object to up performance.

File: cupy/linalg/util.py [hunk 2]
```diff
@@ -26,18 +26,16 @@ def _assert_nd_squareness(*arrays):
...
     return x
 
 
 def _triu(x, k=0):
-    m, n = x.shape
-    u = cupy.arange(m).reshape(m, 1)
-    v = cupy.arange(n).reshape(1, n)
-    mask = v - u >= k
-    x *= mask
+    cupy.ElementwiseKernel(
```
Comment (okuta) [inline cupy/linalg/util.py]: ditto

**Comment by rezoo** [comment]
Thanks. I've fixed.

**Comment by gwtnb** [comment]
LGTM

**Pull request merged**